### PR TITLE
updated eip-1679 to contain eip-1108

### PR DESCRIPTION
**Pull request by zac-williamson**
This is a PR to add eip-1108 to the list of proposed EIPs for Istanbul

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 1679 requires approval from one of (@5chdn, @axic)

File: EIPS/eip-1679.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
     [pending audit](https://medium.com/ethereum-cat-herders/progpow-audit-goals-expectations-75bb902a1f01),
     above and beyond standard security considerations, that should be evaluated
     prior to inclusion.
+- [EIP-1108](https://eips.ethereum.org/EIPS/eip-1108): Reduce alt_bn128 precompile gas costs  
+  - There is currently an outstanding [pull request](https://github.com/ethereum/EIPs/pull/1987) that updates the gas costs to the latest benchmarks
```
Comment (axic) [inline EIPS/eip-1679.md]: I'd prefer we merge #1987 first, remove this comment and merge this to avoid the extra PRs needed to remove this line.

Comment (Shadowfiend) [reply EIPS/eip-1679.md]: Just merged that guy, so it looks like @zac-williamson you just need to drop the note on line 38 and we're maybe good to go here?

Comment (axic) [reply EIPS/eip-1679.md]: @zac-williamson can you please do that? I'd like to merge this.

**Comment by zac-williamson** [comment]
Heya! Sorry for the slow response, I've removed the superfluous line

**Pull request merged**